backvendor/display_test.go

```diff
@@ -0,0 +1,70 @@
+package backvendor
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+)
+
+func TestDisplayDefault(t *testing.T) {
+	const expected string = "@4309345093405934509 =v0.0.1 ~v0.0.0.20181785"
```

Comment:
You can remove `string` from this declaration (and the other similar ones) as the type is inferred from the value.